docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -1,91 +1,75 @@
 git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-mobile.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-portal.git
 ----
 
-Previous versions are available through branches named with the `release-\{semver\}` convention.
-For example, in order to checkout version `0.0.1` of `raincatcher-server`, run the following commands after the cloning:
+Tip: Cloning the repositories provides the latest version. You can use the `git checkout `release-\{semver\}` command if you want a previous version. For example, `git checkout `release-0.0.1`.
 
+2&#46; Download the NPM dependencies for each Git repostory.
 [source,bash]
 ----
-cd raincatcher-server/
-git checkout release-0.0.1
+cd /raincatcher-server
+npm install
 ----
 
-[id='{context}-published-repositories-installing-repository-dependencies']
-[discrete]
-== Installing Repository Dependencies
-
-For each repository previously cloned, run `npm install` so regular NPM dependencies are downloaded.
-
-[id='{context}-published-repositories-starting-mongodb-and-redis']
-[discrete]
-== Starting MongoDB and Redis
-
-. Start MongoDB. For more information, see link:https://docs.mongodb.com/manual/tutorial/manage-mongodb-processes/#start-mongod-processes[Start MongoDB Processes].
-
-. Start Redis. For more information, see link:https://redis.io/topics/quickstart#starting-redis[Starting Redis].
+3&#46; Ensure MongoDB and Redis are running.
 
-[id='{context}-published-repositories-starting-the-core-server']
-[discrete]
-== Starting the Core Server
+For more information, see:
+. MongoDB: https://docs.mongodb.com/manual/tutorial/manage-mongodb-processes/#start-mongod-processes[Start MongoDB Processes]
+. Redis: https://redis.io/topics/quickstart#starting-redis[Starting Redis]
 
+4&#46; Start the Core server.
 [source,bash]
 ----
 cd raincatcher-server/
 npm run start
 ----
 
-NOTE: The terminal is out of use while the Core Server is running.
-
-[id='{context}-published-repositories-running-the-mobile-and-portal-applications']
-[discrete]
-== Running the Mobile and Portal Applications
-
-Open a new terminal and execute the following command in folder `raincatcher-portal/` and also in folder `raincatcher-mobile/`:
+Note: After you start each {PROJECT_NAME} application, it continues to run in the terminal. Closing the terminal stops the application. You must start the Mobile and Portal applications from separate terminals.
 
+5&#46; Start the Mobile application.
 [source,bash]
 ----
+cd raincatcher-mobile/
 npm run start
 ----
 
-Both the Mobile and Portal Applications open automatically in a browser.
-
-== Credentials for the Mobile and Portal Demo Application
-
-Login using the following credentials:
+6&#46; Start the Portal application.
+[source,bash]
+----
+cd raincatcher-portal/
+npm run start
+----
 
-.Credentials for the Mobile and Portal Applications
+The Mobile and Portal applications automatically open in a browser.
```

Comment:
It may be worth to support situation when this will not work and put application urls here.

Reply:
yep. will include the urls.